### PR TITLE
Add harvest command for local environment.

### PR DESCRIPTION
**Pull request by avdata99**
Required to test locally the harvester in a local environment

### CODE REVIEW

**Comment by mogul** [comment]
@avdata99 looks like the Snyk failure expired... Do you want to refresh this branch and re-request review?

**Comment by avdata99** [comment]
> @avdata99 looks like the Snyk failure expired... Do you want to refresh this branch and re-request review?

Done @mogul

**Comment by FuhuXia** [comment]
@adborden fetch and gather consumers used to have their own service running when we do `docker-compose up`, so we don't have to run the `make` command to test harvester workers. But I see they were recently removed by [this commit](https://github.com/GSA/catalog-app/commit/2f82c3efffa2f58d85af8ed0de9b2c1ddebf7053#diff-4e5e90c6228fd48698d074241c2ba760) as unused services. Any reason behind it?

File: Makefile [hunk 1]
```diff
@@ -28,3 +28,20 @@ update-dependencies:
...
+	docker-compose -f docker-compose.yml \
+		-f docker-compose.local.yml \
+		exec app bash -c "source /usr/lib/ckan/bin/activate && \
+			paster --plugin=ckanext-harvest harvester gather_consumer -c /etc/ckan/production.ini"
+
+harvets-fetch-local:
+	docker-compose -f docker-compose.yml \
+		-f docker-compose.local.yml \
+		exec app bash -c "source /usr/lib/ckan/bin/activate && \
+			paster --plugin=ckanext-harvest harvester fetch_consumer -c /etc/ckan/production.ini"
```
Comment (avdata99) [inline Makefile:42]:
```suggestion
	docker-compose -f docker-compose.yml \
		-f docker-compose.local.yml \
		exec app ckan --plugin=ckanext-harvest harvester fetch_consumer
```

File: Makefile [hunk 2]
```diff
@@ -28,3 +28,20 @@ update-dependencies:
 
 up:
 	docker-compose up
+
+harvest-gather-local:
+	docker-compose -f docker-compose.yml \
+		-f docker-compose.local.yml \
+		exec app bash -c "source /usr/lib/ckan/bin/activate && \
+			paster --plugin=ckanext-harvest harvester gather_consumer -c /etc/ckan/production.ini"
```
Comment (avdata99) [inline Makefile:36]:
```suggestion
	docker-compose -f docker-compose.yml \
		-f docker-compose.local.yml \
		exec app ckan --plugin=ckanext-harvest harvester gather_consumer
```

**Comment by adborden** [comment]
@avdata99 just curious, how are the harvest commands handled in catalog-next? I don't see the same commands in the makefile.

**Comment by avdata99** [comment]
@FuhuXia @adborden we don't really need this. When I need to harvest a source in my local environment (in the old or the new catalog) I run these commands manually. We can dismiss this PR.

**Comment by FuhuXia** [comment]
@avdata99 I see. Without `fetch` and `gather` consumers running as a service, and a scheduler service running the `harvester run` command, this PR is a good addition to the `make` command so users have a way to run the harvester workers.

**Pull request merged**